requirements/base.txt

```diff
@@ -9,7 +9,7 @@ certifi==2020.4.5.1       # via elastic-apm, requests, sentry-sdk
 chardet==3.0.4            # via requests
 coreapi==2.3.3            # via drf-yasg
 coreschema==0.0.4         # via coreapi, drf-yasg
-django-admin-index==1.3.0  # via -r requirements/base.in
+django-admin-index==1.4.0  # via -r requirements/base.in
```

Comment:
It looks like this `admin_index` version has conflicting migrations in it. I think it's better to resolve them before using it

Reply:
can you be more specific? I don't see it?

Reply:
Sorry, it was a problem on my side.